packages/onetwopunch/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='onetwopunch'
+pkgver=17.d4ab4e8
+pkgrel=1
+pkgdesc='Use unicornscan to quickly scan all open ports, and then pass the open ports to nmap for detailed scans.'
+groups=('blackarch' 'blackarch-scanner')
+arch=('any')
+url='https://github.com/superkojiman/onetwopunch'
+license=('MIT')
+depends=('nmap' 'unicornscan')
+makedepends=('git')
+source=('git+https://github.com/superkojiman/onetwopunch.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/onetwopunch"
+  
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
The package provides a tag (release), it should be also in `$pkgver`. Use the following command instead.

```
git describe --long | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```